src/sentry/tsdb/redis.py

```diff
@@ -41,10 +47,10 @@ class SuppressionWrapper:
     def __init__(self, wrapped):
         self.wrapped = wrapped
 
-    def __enter__(self):
+    def __enter__(self) -> Any:
         return self.wrapped.__enter__()
```

Comment:
could probably make this typesafe with `wrapped: ContextManager[T]` and have `__enter__` return `T` and have this class extend `Generic[T]`

Reply:
Thanks! This is what I was looking for.